# data = {1,2,3}
# for i in data:
#     data.remove(i)
# print(data)
# thrown an error due to anacondas strict nature to handle exceptions and if it was visual studio it would show output 2



